tests/tests/fee_tests.cpp

```diff
@@ -174,12 +174,12 @@ BOOST_AUTO_TEST_CASE(asset_claim_fees_test)
 
       }
 
-      if( db.head_block_time() <= HARDFORK_413_TIME )
+      if( db.head_block_time() <= HARDFORK_CORE_188_TIME )
```

Comment:
The change here and a few lines below are wrong.
If we decide to remove the check about HARDFORK_413_TIME, the unit test code here should be updated accordingly.

Reply:
Fixed